ccloader/js/ccloader.js

```diff
@@ -113,14 +109,93 @@ export class ModLoader {
 		}
 	}
 
+
+	_sortModByExtension(manifestPath, modFiles, ccmodFiles, packedMods) {
+		const endings = this.filemanager.endings;
+		const selectedArray = [
+			modFiles,
+			ccmodFiles,
+			packedMods
+		];
```

Comment:
It's nice that you reused the endings from `filemanager` but in the next line you hardcoded it anyway. This is actually worse since it can become out of sync without anyone noticing. Hardcoding `endings` would allow you to implement this method with a switch which would make this method way easier to understand.